translationCoreApps/translationWords_Check_plugin
# Update source

#### This pull request addresses:

This PR addresses https://github.com/unfoldingWord-dev/translationCore/issues/875

It updates the sources of the wordlist by using the UW Api. 


#### How to test this pull request:
There isn't exactly a way to test this, maybe switch around checks, make sure there isn't any glaring issues.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationwords_check_plugin/47)
